core/store/src/main/java/uk/gov/gchq/gaffer/store/Store.java

```diff
@@ -257,7 +260,7 @@ public static Store createStore(final String graphId, final Schema schema, final
                     .asSubclass(Store.class)
                     .newInstance();
         } catch (final InstantiationException | IllegalAccessException | ClassNotFoundException e) {
-            throw new IllegalArgumentException("Could not create store of type: " + storeClass, e);
+            throw new IllegalArgumentException(String.format("Could not create store of type: %s", storeClass), e);
```

Comment:
I do like this change but it does make it harder to review the actual changes when there are so many unrelated formatting changes